### ISSUE TITLE
v1.1.319 appears to be broken on Node.js v12

### ISSUE THREAD

**Issue by RobertCraigie** [microsoft/pyright#5582]
**Describe the bug**

This error occurs in the Python wrapper:
```
2023-07-26 08:53:32,354 - DEBUG - pyright.node - Running node command with args: ['/opt/hostedtoolcache/node/12.22.12/x64/bin/node', '/home/runner/.cache/pyright-python/1.1.319/node_modules/pyright/index.js', '--version']
internal/modules/cjs/loader.js:818
  throw err;
  ^

Error: Cannot find module 'node:process'
Require stack:
- /home/runner/.cache/pyright-python/1.1.319/node_modules/pyright/dist/pyright.js
- /home/runner/.cache/pyright-python/1.1.319/node_modules/pyright/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
    at Function.Module._load (internal/modules/cjs/loader.js:667:27)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.7742 (/home/runner/.cache/pyright-python/1.1.319/node_modules/pyright/dist/pyright.js:1:390)
    at i (/home/runner/.cache/pyright-python/1.1.319/node_modules/pyright/dist/pyright.js:1:903)
    at Object.5[265](https://github.com/RobertCraigie/pyright-python/actions/runs/5666695204/job/15353963147?pr=197#step:8:266) (/home/runner/.cache/pyright-python/1.1.319/node_modules/pyright/dist/vendor.js:2:774786)
    at i (/home/runner/.cache/pyright-python/1.1.319/node_modules/pyright/dist/pyright.js:1:903)
    at Object.3482 (/home/runner/.cache/pyright-python/1.1.319/node_modules/pyright/dist/pyright-internal.js:1:1370625)
    at i (/home/runner/.cache/pyright-python/1.1.319/node_modules/pyright/dist/pyright.js:1:903) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/runner/.cache/pyright-python/1.1.319/node_modules/pyright/dist/pyright.js',
    '/home/runner/.cache/pyright-python/1.1.319/node_modules/pyright/index.js'
  ]
}
```

Link to [CI run](https://github.com/RobertCraigie/pyright-python/actions/runs/5666695204/job/15353963147?pr=197). Jobs running Node.js v14 & v16 passed successfully.

Here's the `package.json` we use in case that's helpful: https://github.com/RobertCraigie/pyright-python/blob/7373ff146a51507ec75c463b8dc57c63cd762d4d/pyright/_utils.py#L15C1-L23C2

**Code or Screenshots**

N/A

**VS Code extension or command-line**

Command-line v1.1.319

**Comment by erictraut** [comment]
Thanks for the bug report.

I can't think of anything that I changed in the past week that would have affected node compatibility.

@debonte, @rchiodo, @heejaechang, do you know of anything that came in from pylance that would have had this effect? Could one of you dig into this? I'm traveling today and won't be able to investigate until tomorrow at the earliest.

Node v12 is _very_ old, so if there's a good reason to no longer support it, we could consider bumping the minimum requirements to something newer like v14.

**Comment by erictraut** [comment]
This break was due to a version update to the `chalk` library. It came in through the weekly pylance code sync. I've reverted `chalk` from 5.x back to 4.x to retain node v12 compatibility for now.

If pyright users are still using node v12, I highly encourage them to upgrade to at least v14. They will see a 2x speed boost in doing so.

Also, we will likely remove support for v12 in the near future.

**Comment by debonte** [comment]
> I've reverted chalk from 5.x back to 4.x to retain node v12 compatibility for now.

@heejaechang, the `chalk` upgrade came from https://github.com/microsoft/pyrx/pull/3889. Any concerns about rolling it back? For example, was this a security fix?

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.320, which I just published.
